config/prod/htan-dcc-center-a.yaml

```diff
@@ -20,11 +20,13 @@ parameters:
     - "arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/adam.taylor@sagebase.org"
     - "arn:aws:iam::728882028485:role/htan-project-TowerForgeBatchHeadJobRole-17MIZHWBA1TC5"
     - "arn:aws:iam::728882028485:role/htan-project-TowerForgeBatchWorkJobRole-17360PQARGWYU"
+    - "arn:aws:sts::728882028485:assumed-role/AWSReservedSSO_TowerViewer_fd6aee98ec127705/adam.taylor@sagebase.org"
   DenyDeleteARNs:
     - "arn:aws:iam::070278699608:user/inodb"
     - "arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/adam.taylor@sagebase.org"
     - "arn:aws:iam::728882028485:role/htan-project-TowerForgeBatchHeadJobRole-17MIZHWBA1TC5"
     - "arn:aws:iam::728882028485:role/htan-project-TowerForgeBatchWorkJobRole-17360PQARGWYU"
+    - "arn:aws:sts::728882028485:assumed-role/AWSReservedSSO_TowerViewer_fd6aee98ec127705/adam.taylor@sagebase.org"
```

Comment:
this is deny you from deleting stuff in the bucket.  did you really want to restrict yourself from deleting stuff?

Reply:
Yes, we have typically restricted delete for all users of HTAN buckets (including internal users) except for the two admin roles (@clarisse-lau  and @thomasyu888). Adding my TowerViewer role to DenyDelete was intentional.